addons/mail/static/src/models/link_preview_image_view.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'LinkPreviewImageView',
+    recordMethods: {
+        /**
+         * Handles mouse enter event for the container of this element.
+         */
+        onMouseEnter() {
+            this.update({ isHovered: true });
+        },
+        /**
+         * Handles mouse leave event for the container of this element.
+         */
+        onMouseLeave() {
+            this.update({ isHovered: false });
+        },
+        /**
+         * @private
+         * @returns {LinkPreviewAsideView|FieldCommand}
+         */
+        _computeLinkPreviewAsideView() {
+            if (!this.linkPreview.isDeletable) {
+                return clear();
+            }
+            if (this.messaging.device.isMobileDevice) {
+                return {};
+            }
+            if (this.isHovered || (this.linkPreviewAsideView && this.linkPreviewAsideView.linkPreviewDeleteConfirmDialog)) {
+                return {};
+            }
+            return clear();
+        },
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeImageUrl() {
+            return this.linkPreview.og_image ? this.linkPreview.og_image : this.linkPreview.source_url;
```

Comment:
This entire segmentation seems wildly inconsistent. Why is some image stuff being predicated upon `image_mimetype`, other on `og_mimetype`, and yet other on `og_image`?

Reply:
See previous comment